### PR TITLE
Ruff updates

### PR DESCRIPTION
**Pull request by justmobilize**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,42 +1,21 @@
-# SPDX-FileCopyrightText: 2020 Diego Elio Pettenò
+# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries
```
Comment (jepler) [inline .pre-commit-config.yaml:1]: unless you consider this a complete rewrite, the original author attribution must remain.

**Review by jepler — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (dhalbert) [reply .pre-commit-config.yaml:1]: Also, since you are not under contract for this work, you can omit "for Adafruit Industries"

Comment (justmobilize) [reply .pre-commit-config.yaml:1]: Can totally update.

Would you actually keep these PRs or just recreate them? I made these so we could look and see what was the right path.

And haven't received a full explanation on when the `for Adafruit` should be there. In my opinion I'm doing it for Adafruit, whether in contract or not...

**Review by FoamyGuy — APPROVED** [review]
I think this is ready to go. 

The latest commits address the prior requested feedback items.

I tested this version of PyPortal as well as the Ruff version of PortalBase and Requests all together on a PyPortal Titano with all 3 example scripts in this repo.

**Pull request merged**